### ISSUE TITLE
development menu does not open on real device Android

### ISSUE THREAD

**Issue by kimvex** [facebook/react-native#15917]
The development menu does not display when you hit the radio button on my real device

"react-native": "0.47.2"
Android 7.0

**Comment by edudel** [comment]
linked to this issue
https://github.com/facebook/react-native/issues/6932
**HOWEVER**,
I can't see the dev menu yet (android). IOS works
RN 0.48.3. (newest)
how can I get the real time changes after using the bundle ?
I got this after upgrading react
Unable to load script from assets 'index.android.bundle'. Make sure your bundle is packaged correctly or you're running a packager server.
so I had to do execute
react-native bundle --platform android --dev true --entry-file index.android.js .........
etc
now the dev menu is gone in android.. even when I run react-native run-android, react-native run-android --variant=debug etc etc

Any ideas ?
